### ISSUE TITLE
Binary releases?

### ISSUE THREAD

**Issue by coolaj86** [dotenv-rs/dotenv#64]
It would be really awesome to have this as a binary that could run like this:

```bash
dotenv .env -- someprogram --foo bar
```

Do you have any interest in supporting that? I'd be willing to help.

**Comment by shesek** [comment]
Why not `(source .env && someprogram --foo bar)`? (requires using `export` in your `.env` files to make them shell-compatible, which `dotenv` accepts too. use `.` instead of `source` if you're using `sh`.)

**Comment by coolaj86** [comment]
1. It's not a cross-platform approach.
2. It requires a shell, which you don't always have with service managers like systemd.

**Comment by mhutter** [comment]
>It's not a cross-platform approach.

That's true but every platform offers built-in tools to read ENV vars from a file

>It requires a shell, which you don't always have with service managers like systemd.

Most service manager already support reading env vars from a file. For systemd, there is [`EnvironmentFile`](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=)
